### ISSUE TITLE
Option "Skip debugging options dialog (use Ctrl key to override)" does not work when turned on

### ISSUE THREAD

**Issue by asb-capfan** [Komodo/KomodoEdit#149]
This is Komodo IDE, version 9.0.0, build 87165, platform win32-x86. Built on Wed Mar 18 21:58:08 2015.

When I check the debugger option "Skip debugging options dialog (use Ctrl key to override)" in Edit > Preferences > Debugger and run a Perl Script using Crtl + F5, no debug options dialog appears. 
This worked fine in Komodo IDE 8.x

**Comment by Naatan** [comment]
I'm confused by your description, the way I am reading your message it seems like you are saying you were expecting to see an option dialog when executing a menu entry that explicitly states it will not show an option dialog? Could you rephrase your issue? Thanks :)

**Comment by asb-capfan** [comment]
Sure, here you go.
The option does say:
a) skip debugging options, and
b) but if you press Ctrl + F5, then show it - that's the part in the brackets: Skip debugging options dialog (**use Ctrl key to override**)

**Comment by Naatan** [comment]
I think the "use Ctrl key to override" refers to clicking the button in your toolbar, it would not adjust what the keybinding is. The keybinding is as you define it, or as the default is defined.

Regardless, it seems when I enable this setting and press the toolbar button with Ctrl I still do not get an options window, so will target this for 9.1.

**Comment by asb-capfan** [comment]
FYI: The default key binding scheme defines Ctrl + F5 as "Debugger: Go/Continue".

**Comment by toddw-as** [comment]
This command has changed in Komodo IDE 9 - instead of using Ctrl to override the behaviour, there are now two separate commands (*Go/Continue* - "F5" by default, and *Go/Continue - skip the Options dialog*). The latter was meant to be set to "Ctrl+F5" by default - but I see that is not the case, so I'll need to fix that.

The skip debugger preference is just meant to swap these commands around, so F5 becomes the *Go/Continue - skip the Options dialog* command.

**Comment by toddw-as** [comment]
Fixed in commits - target build: 9.0.1

**Comment by Naatan** [comment]
So this is now basically just a pref to switch they keybinding around? That seems awkward, should just remove this pref altogether.

**Comment by toddw-as** [comment]
Yes - that was my original intention (to remove the pref altogether), but that would still needs some further work to make it discoverable - such as adding a menu entry for the run/skip debugging dialog.
